CMakeLists.txt

```diff
@@ -24,6 +25,10 @@ if (DETRAY_ARRAY_PLUGIN)
     set(ALGEBRA_PLUGIN_INCLUDE_ARRAY ON)
 endif()
 
+if (DETRAY_VECMEM_ARRAY_PLUGIN)
+    set(ALGEBRA_PLUGIN_INCLUDE_VECMEM ON)
```

Comment:
The ALGEBRA_PLUGIN_INCLUDE_VECMEM flag is meant to build vecmem::static_array as the algebra type itself, i.e. as full, distinct plugin. I want this for testing mainly. So, if you want to use matrices made of vecmem::static_array you set ALGEBRA_PLUGIN_INCLUDE_VECMEM (which won't work with detray right now), but if you want to use vecmem::static_array and vecmem::vector as global container types in detray (i.e. the containers for the matrices), you set ALGEBRA_PLUGIN_USE_VECMEM instead. I will simplify this, when I'm done testing. Then, there won't be distinct array plugins anymore. Sorry for the mess!
I would therefore propose to call this DETRAY_USE_VECMEM, which then sets ALGEBRA_PLUGIN_USE_VECMEM.

Reply:
How should I change here?

Reply:
okay fixed now